### ISSUE TITLE
Bluescreen on closing browser

### ISSUE THREAD

**Issue by jeanluc162** [Feodor2/Mypal68#158]
I've recently experienced a bluescreen when I closed the browser. The only tab I had opened was youtube.
[2022_09_27_Crash List.html.zip](https://github.com/Feodor2/Mypal68/files/9668000/2022_09_27_Crash.List.html.zip)

**Comment by IP-CAM** [comment]
Such frequently happen, MyPal68 is still far from working error free.

**Comment by jeanluc162** [comment]
I am aware, I just thought every Bluescreen report helps

**Comment by NEO-SPECTRUMAN** [comment]
а я вот сколько пользуюсь бсодов так и не словил

**Comment by ScratchApps** [comment]
Я тоже



________________________________
От: NS ***@***.***>
Отправлено: среда, 28 сентября 2022 г., 23:34
Кому: Feodor2/Mypal68 ***@***.***>
Копия: Subscribed ***@***.***>
Тема: Re: [Feodor2/Mypal68] Bluescreen on closing browser (Issue #158)


а я вот сколько пользуюсь бсодов так и не словил

**Comment by chrischris888** [comment]
it does not work already for one year ... it is impossible to work. I don't believe it will ever work.
What I am using right now is 360chrome.exe for Windows XP version 360EE_12.0.1592.0_Modified
All others will not work ... 13.0 will not work, 11.0 will not open, 14.0 will not work, 15.0 will not work.

**Comment by andika207** [comment]
@jeanluc162 this issue is normally associated to chrome browsers sometimes when you close either the browser or a tag it pops a BSOD up 
not sure why but I also suffered from this with Centaury or Mypal 68 and the report is similar to yours, ntoskrnl, dxg, hall files are involved in the case (PAE patch ?)

the old Firefox used to crash before it became stable with FF52 but I don't remember having a single BSOD
same thing applies to Serpent 52 @roytam1 it's pretty stable, as per my experience

**Comment by NEO-SPECTRUMAN** [comment]
serpent52 is unstable too

**Comment by GreenReaper** [comment]
Ultimately browsers shouldn't be able to cause a bluescreen, it is beyond their power as an application - the issue will likely be your video drivers, which are of course used for handling video (and graphics in general). Probably they didn't handle memory correctly but it only became clear when the browser released it all at the end. You could try tweaking the various 'gfx' options in about:config to workaround the problem by not using your graphics hardware in the same way.

**Comment by GreenReaper** [comment]
They shouldn't, and if they do the actual issue is still the driver; but if the browser triggers a flaw in the driver, it may be possible to avoid that.

**Comment by andika207** [comment]
if all other browsers are running flawlessly, and there's one that triggers a crash then this browser is the culprit. end of story. 
don't try to blame the drivers or whatever else.

**Comment by GreenReaper** [comment]
I do; I will. Since Windows NT/2000/XP, only drivers and other software installed with kernel mode access (including the kernel itself) can cause that kind of error. If they do so in combination with their use by other software, it remains the responsibility of the driver or kernel because they're the ones with access. Usually there is an issue with their logic or sequence of actions that only reveals itself in certain situations.

Firefox actually has a big list of drivers that it refuses to use certain accelerations on because it knows that they are flawed and it will get blamed for their issues, but that doesn't mean it is responsible for the issues.

**Comment by MWF2** [comment]
for information about bsod, go to issue 3 that i made, is where you can tell about send report, if is hardware or is win32k #3

**Comment by Feodor2** [comment]
Let further discuss on #3
Tell there on model of the cpu, i see likely it happens mostly on newer 'I' cpus, but i do not have any this type cpu machines, and don't know how to reproduce.

**Comment by andika207** [comment]
@NEO-SPECTRUMAN Serpent 52 is pretty stable to me. @roytam1
the downside is it's terribly slow because it's based on the crap Basilisk @mattatobin
go to 4chan website and start taping on multiple webms in a row, that crap Basilisk will freeze righ away LOL
everytime I launch Serpent 52 (more than 100 tags at once) the CPU load goes up and stays at around 30%-50%
this is not normal because unlike chrome only 1 tab gets loaded on startup
maybe it was not intended for such task and I should keep no more than 20 open tags at a time
further more, memory consumption is another flaw in comparison to chrome because it won't drop when you close tags if you ever reach RAM limit
hardware quality or Windows version are not a possible cause as it sturggles on both ways.
the original Firefox 52 runs better than either of them
I am seriously thinking on OCA @Skulltrail192 if I get it to work with not much hassle I won't be using these deprecated browsers anymore.
in short: the 360EE fork runs much smoother in XP than the crap BASILISK does under W7
it's hard to believe but it's a fact

**Comment by chrischris888** [comment]
I am using right now 

360chrome.exe for Windows XP version 360EE_12.0.1592.0_Modified 

already for more then 5 months, and I can open dozens of tabs, YouTube's, Facebook, and the Messenger which now all works without any problem. I am using this browser for everything except one single website, which work only on MyPal 29.3, but I am extremely satisfied. I am no longer trying MyPal 68 or any other related, which they don't work. In these 5 months I never had any blue screen with the 

 360chrome.exe for Windows XP version 360EE_12.0.1592.0_Modified

and it never crashed, but it notifies me when there are too less RAM to clear the browsing data and also optimize memory (two buttons on the bottom of the browser).

This is the best browser ever, for Windows XP SP3 Media Center Edition 2005.

I am mostly sincere and honest that this really WORKS !

**Comment by NEO-SPECTRUMAN** [comment]
> I won't be using these deprecated browsers anymore.

therre is no old addons on "new" not deprecated browsers
only bullshit

**Comment by NEO-SPECTRUMAN** [comment]
> the original Firefox 52 runs better than either of them

is that because it doesn't support tons of new js shit?

**Comment by chrischris888** [comment]
LINK FOR  360chrome.exe for Windows XP sp3 version 360EE_12.0.1592.0_Modified
https://msfn.org/board/topic/182876-360-extreme-explorer-modified-version/page/4/

**Comment by andika207** [comment]
@chrischris888 LOL https://github.com/Feodor2/Mypal68/issues/3#issuecomment-1126878320

**Comment by andika207** [comment]
@NEO-SPECTRUMAN Basilisk/Serpent garbage is not based on FF52 thus it struggles so much.
I mostly use it because of the ''Firefox 2'' theme (skin) which is no longer available for unknown reasons @mattatobin
the multi process mode helps a little but doesn't equal the performance of the original Firefox at all
however on Centaury browser I need ''Classic Theme Restorer'' to match the FF2/3 skin layout https://github.com/Feodor2/Centaury/issues/50

**Comment by mattatobin** [comment]
Did Moonchild not re-assign Ryan's themes back to Ryan before removing my other content? A shame. However, those themes I wanted to take over will return to the net never the less just under different names and IDs and a double check on the licenses.

He could restore them though if he can figure out how. I know how. Might even say if asked. Luckily he has every backup I have from 2017 onward, I insisted on it.

**Comment by andika207** [comment]
I don't actually use moonchild's browsers @wolfbeast 
even so the FF2 and FF3 style themes have always been my favorite ones 
https://addons.palemoon.org/addon/moonfox3/
https://addons.palemoon.org/addon/fox-2-the-moon/

this one is still compatible with Serpent 52 thus I don't get why it's been removed from Basilisk website
https://rctgamer3.github.io/ff2-default/

**Comment by mattatobin** [comment]
Post-Australis CTR is needed for most pre-Australis themes to be given the best possible chance of working. That was a big part in why Aris created it even if not the inital goal. Providing the UI elements the skin is expected to style when applied always works better than what little remaining that can be matched by id or class or another selector.

Regardless, @andika207 it isn't very good form to start highlighting unaffiliated people that haven't a need to even be involved let alone had any expectation of being pinged. I would strongly suggest you cut down on that tendancy. I am getting to the point where it doesn't bother me much and perhaps if I just respond if I think I can add anything not to exclusively harrass someone who got my attention. But others aren't at that point in their lives if they can ever manage to reach it in the first place and they may turn a rabid eye to you if you @ mention someone one too many times.

It doesn't take much to piss someone off enough that they start a crusade with their friends ;)
